actions/email.users.with.decom.flavors.yaml

```diff
@@ -47,6 +47,11 @@ parameters:
     description: "tick to search in all projects - default True"
     required: true
     default: true
+  use_override:
+    type: boolean
+    description: "TESTING: Set this flag so ALL emails will be redirected to override email"
```

Comment:
Nit: can we drop `TESTING` it makes it look this doesn't work

Also don't we have override email? If so we should just use that instead of another bool for an optional field

Reply:
`override_email_address` is the email we use when a VM doesn't have a email set 
`use-override` is a flag which makes it so every email gets sent to one set in `override_email_address`